src/kedro_vizro_labeling/dashboards/iris/app.py

```diff
@@ -50,11 +53,17 @@ def show_plot():
             text="Show histogram",
             actions=[
                 ProtectedAction(
+                    # AM question: is this actually how you want the flow to work or is it just a workaround? Compared
+                    # to my example where the plot is protected by the @capture("graph") and there's no button to
+                    # enable it.
```

Comment:
Just a way to showcase `protected` and trigger a modal. It seemed natural to me as dash-auth as a `protected_callback` but I don't have a use case in mind. 

It would have been better to write a `ProtectedPage` as I also have a real use case for it.